FBC/ProductsAndServices/FinancialProductsAndServices.rdf

```diff
@@ -985,15 +986,16 @@
 		<rdfs:subClassOf rdf:resource="&fibo-fbc-pas-fpas;TradeIdentifier"/>
```

Comment:
we say earlier that a Trade may consist of many Transactions (UTIs) and so saying UTI is a subclass of TradeIdentifier does not seem consistent

Reply:
It is.  In many cases there is only one transaction, in which case they would be the same identifier.  It's only in cases where there are multiple transactions comprising a trade when they are many to 1.

Reply:
@tahoeblue Thanks Jeff, That's how I understood it.  The places in Nordea's process where there might be more than one UTI involved in the 'trade' side of a deal (the customer facing side) seemed to involve either multiple bonds (one of which might be zero coupon) or structured products, or other more complex deals, some of which involved more than one counterparty.  They also had an internal "bank side" of a deal, where they hedged risk internally, but it was all part of the same overarching "deal".  There were many more internal steps that would not count as separate "transactions" as you've outlined.